docs/docsite/rst/networking_guide/network_debug_troubleshooting.rst

```diff
@@ -138,6 +138,64 @@ Then review the log file and find the relevant error message in the rest of this
 
 .. For details on other ways to authenticate, see LINKTOAUTHHOWTODOCS.
 
+.. _socket_path_issue:
+
+Category "socket_path issue"
+========================================================
+
+**Platforms:** Any
+
+The ``socket_path does not exist or cannot be found``  and ``unable to connect to socket`` message is new in Ansible 2.5. This message means that
+socket used the talk to the remote network device does not exist or unable to connect to.
+
+
+For example:
+
+.. code-block:: none
+
+   fatal: [spine02]: FAILED! => {
+       "changed": false,
+       "failed": true,
+       "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/connection.py\", line 115, in _exec_jsonrpc\nansible.module_utils.connection.ConnectionError: socket_path does not exist or cannot be found\n",
+       "module_stdout": "",
+       "msg": "MODULE FAILURE",
+       "rc": 1
+   }
+
+or
+
+.. code-block:: none
+
+   fatal: [spine02]: FAILED! => {
+       "changed": false,
+       "failed": true,
+       "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_TSqk5J/ansible_modlib.zip/ansible/module_utils/connection.py\", line 123, in _exec_jsonrpc\nansible.module_utils.connection.ConnectionError: unable to connect to socket\n",
+       "module_stdout": "",
+       "msg": "MODULE FAILURE",
+       "rc": 1
+   }
+
+Suggestions to resolve:
+
+Follow the steps detailed in enable_network_logging_.
```

Comment:
May be GitHub formatting issue, though this doesn't look right.
I think you want
```
 :ref:`enable network logging <enable_network_logging>`
```